modules/pel/peltool/peltool.py

```diff
@@ -528,6 +528,7 @@ def printPELCount(path: str, config: Config, extension: str):
 
```

Comment:
Commit Message:

Your PR description can be included in the commit message. We primarily look for these details in the commit message.